### ISSUE TITLE
Can't find ADIOS1 dependency on Titan, error message is ambiguous

### ISSUE THREAD

**Issue by williamfgc** [ornladios/ADIOS2#182]
build_gnu> cmake -DCMAKE_BUILD_TYPE=Release -DADIOS2_USE_BZip2=OFF -DADIOS1_DIR=/sw/xk6/adios/1.12.0/cle5.2_gnu4.9.3 ../ADIOS2
-- The C compiler identification is GNU 4.9.3
-- The CXX compiler identification is GNU 4.9.3
-- Cray Programming Environment 2.5.9 C
-- Check for working C compiler: /opt/cray/craype/2.5.9/bin/cc
-- Check for working C compiler: /opt/cray/craype/2.5.9/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Cray Programming Environment 2.5.9 CXX
-- Check for working CXX compiler: /opt/cray/craype/2.5.9/bin/CC
-- Check for working CXX compiler: /opt/cray/craype/2.5.9/bin/CC -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find ZFP (missing:  ZFP_LIBRARY ZFP_INCLUDE_DIR) 
-- Found MPI_C: /opt/cray/craype/2.5.9/bin/cc  
-- Found MPI_CXX: /opt/cray/craype/2.5.9/bin/CC  
-- Could NOT find ZeroMQ (missing:  ZeroMQ_LIBRARY ZeroMQ_INCLUDE_DIR) 
-- Could NOT find HDF5 (missing:  HDF5_LIBRARIES HDF5_INCLUDE_DIRS C) 
[Sun Jun 25 07:48:40 2017] [unknown] Fatal error in MPI_Init: Other MPI error, error stack:
MPIR_Init_thread(525): 
MPID_Init(228).......: channel initialization failed
MPID_Init(611).......:  PMI2 init failed: 1 
/sw/xk6/adios/1.12.0/cle5.2_gnu4.9.3/bin/adios_config: line 242: 29050 Aborted                 ${MYDIR}/adios_list_methods
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - found
-- Found Threads: TRUE  
-- Could NOT find ADIOS1 (missing:  ADIOS1_LIBRARY ADIOS1_INCLUDE_DIR) (found suitable version "1.12.0", minimum required is "1.12.0")

Found or not found?

**Comment by pnorbert** [comment]
I think we only need to process the output of "adios_config -l" to
configure adios1 for adios2. Maybe also -c if necessary.
adios_config by itself runs adios_list_methods which is an MPI binary and
does not run on the login node.
Chuck has found out this before. I thought it was fixed already.

**Comment by williamfgc** [comment]
@chuckatkins Can you confirm? @pnorbert I am trying on Titan with this commit from Friday:   Merge: 230ef75 502b61b
Author: Chuck Atkins <chuck.atkins@kitware.com>
Date:   Fri Jun 23 15:54:25 2017 -0400

    Merge pull request #179 from chuckatkins/add-lib-version-info
    
    Setup library versioning.

Not sure if this has the fix. Thanks!

**Comment by chuckatkins** [comment]
#179 is entirely unrelated. It setup version I fo for libadios2, i.e libadios2.so -> libadios2.so.2.0.0 and libadios2.so.2 -> libadios2.so.2.0.0 .  This issue needs the FindADIOS1.cmake adjusted.

**Comment by williamfgc** [comment]
@chuckatkins My apologies, I was just pointing out the head commit I am working with. Let me rephrase: Not sure if the code has the fix by then. Thanks!

**Comment by chuckatkins** [comment]
Fixed by #186
